src/main/java/core/basesyntax/strategy/handler/PurchaseHandler.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.strategy.handler;
+
+public class PurchaseHandler extends DecrementHandler {
```

Comment:
The `PurchaseHandler` class should handle the 'purchase' operation by checking the result balance before saving it in the Storage. It should throw a `RuntimeException` if the balance is negative, as per the checklist item 'Handling Purchase operation'. Currently, the class is empty and inherits from `DecrementHandler` without any additional implementation. Make sure to override the necessary method to include the balance check logic.